saleor/graphql/product/filters.py

```diff
@@ -314,13 +314,14 @@ def filter_products_by_stock_availability(qs, stock_availability, channel_slug):
     )
     allocated_subquery = Subquery(queryset=allocations, output_field=IntegerField())
 
-    stocks = list(
+    stocks = (
```

Comment:
aren't brackets here unnecessary?

Reply:
We could write this without brackets but usually, that's how we format long lines so I think this is fine, but thanks for keeping an eye on it.